rules/ensure-default-service-accounts-has-only-default-roles/test/failed_clusterrolebinding /input/clusterrolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: read-secrets-global
+subjects:
+- kind: ServiceAccount
+  name: manager
+  apiGroup: rbac.authorization.k8s.io
+- kind: Group
+  name: dev
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
why does this clusterrolebinding fail? It has no SA named default in its subjects
also - SA subjects have fields kind, name and namespace. users and groups have apiGroup instead of namespace

Reply:
fixed